codes/dart/chapter_computational_complexity/worst_best_time_complexity.dart

```diff
@@ -0,0 +1,42 @@
+/**
+ * File: worst_best_time_complexity
+ * Created Time: 2023-02-12
+ * Author: Jefferson (JeffersonHuang77@gmail.com)
+ */
+
+class WorstBestTimeComplexity {
```

Comment:
Remove the class if it is unnecessary.

(In Java, we have to create a class but it is not elegant.)